test/functional/commands/attribute-e2e-specs.js

```diff
@@ -1,66 +1,60 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { retryInterval } from 'asyncbox';
 import { APIDEMOS_CAPS } from '../desired';
 import { initDriver } from '../helpers/session';
 
-
 chai.should();
 chai.use(chaiAsPromised);
 
 let driver;
 let animationEl;
 
-async function waitForElement (strategy, selector) {
-  return await retryInterval(10, 3000, driver.findElOrEls.bind(driver), strategy, selector, false);
-}
-
 describe('apidemo - attributes', function () {
-  before(async () => {
+  before(async function () {
     driver = await initDriver(APIDEMOS_CAPS);
-
-    let animation = await waitForElement('accessibility id', 'Animation');
-    animationEl = animation.ELEMENT;
+    animationEl = await driver.elementByAccessibilityId('Animation');
   });
   after(async () => {
-    await driver.deleteSession();
+    await driver.quit();
   });
   it('should be able to find resourceId attribute', async () => {
-    await driver.getAttribute('resourceId', animationEl).should.eventually.become('android:id/text1');
+    await driver.getAttribute(animationEl, 'resourceId').should.eventually.become('android:id/text1');
```

Comment:
i find it odd that something like `await animationEl.getAttribute('resourceId')` wouldn't work just as well. it seems more idiomatic to me if it does (goes for all the rest like this as well).